omnigan/blocks.py

```diff
@@ -239,6 +239,7 @@ def __init__(
         self,
         n_upsample=4,
         n_res=4,
+        input_dim=2048,
         dim=64,
```

Comment:
can we rename that to something like `projection_dim` or somehting? to make it extra clear?